package.json

```diff
@@ -82,7 +82,7 @@
     "sinon": "1.17.3",
     "sinon-chai": "2.8.0",
     "style-loader": "0.13.1",
-    "webpack": "1.12.14",
+    "webpack": "1.14.0",
```

Comment:
I would say just request some general "smoke testing" of the uploader functionality? (Click around the UI, verify that a couple of devices upload.) It doesn't feel high-risk to me. If the apps builds (which it does), everything's probably fine.